src/metric/kafka.ts

```diff
@@ -1,30 +0,0 @@
-import * as Kafka from 'no-kafka';
-
-import { Configuration, IMetric, Logger } from '../common';
```

Comment:
@arshu16 think the whole metric namespace can be removed if this isn't going to be used?

Reply:
@johndagostino Cool. It isn't used anyway.